src/main/java/com/deque/axe/android/AxeView.java

```diff
@@ -98,11 +100,21 @@
    */
   public final String hintText;
 
+  /**
+   * Direct copy of value.
```

Comment:
You're going to have merge conflicts with the https://github.com/dequelabs/axe-android/pull/80 PR